ting-bill-split.go

```diff
@@ -362,35 +364,166 @@ func createBillsFile(path string) {
 	}
 }
 
-func main() {
-	fmt.Printf("Ting Bill Splitter\n\n")
+func isFileMatch(fileName string, nameTerm string, ext string) bool {
+	r := regexp.MustCompile(`(?i)^[\w-]*` + nameTerm + `[\w-]*$`)
+
+	if ext != "" {
+		r = regexp.MustCompile(`(?i)^[\w-]*` + nameTerm + `[\w-]*(\.` + ext + `)$`)
+	}
+
+	return r.MatchString(fileName)
+}
+
+func parseDir(path string) {
+	files, err := ioutil.ReadDir(path)
```

Comment:
Move this down after the `var` declarations so the error check is right after the function call

Reply:
You ever gotten highlighting like this?  Tests still pass, and now it stays that color anywhere I move it, even if I revert back to where it was per your comment above.

![image](https://user-images.githubusercontent.com/730506/38164907-3b22d8fc-34d1-11e8-92db-94a5ee1102ab.png)